src/helpers/runNotificationListener.ts

```diff
@@ -21,14 +20,37 @@ ketlAttestationContract.on('EntanglementRegistered', async () => {
   }
 })
 
-getFeedsContract.on('CommentAdded', async () => {
-  try {
-    const tokens = await getTokens({ repliesEnabled: true })
-    await sendFirebaseNotification({ tokens })
-  } catch (err) {
-    console.error(err)
+getFeedsContract.on(
+  'CommentAdded',
```

Comment:
Should we listen for reactions as well? To send `hotPost` when it reaches certain amount of reactions?